src/Cache/_ScsDataClient.php

```diff
@@ -430,7 +430,7 @@ public function listErase(string $cacheName, string $listName, ?int $beginIndex
         return new CacheListEraseResponseSuccess();
     }
 
-    public function dictionarySet(string $cacheName, string $dictionaryName, string $field, string $value, bool $refreshTtl, ?int $ttlSeconds = null): CacheDictionarySetResponse
+    public function dictionarySet(string $cacheName, string $dictionaryName, string $field, string $value, bool $refreshTtl, ?int $ttlSeconds = null): CacheDictionarySetFieldResponse
```

Comment:
I'm a little concerned about how these method names no longer match the SCC methods. Having `$this->client->dictionarySetField()` end up calling `dictionarySet()` here seems like it's going to be very confusing for us to maintain in the future. I'd propose we go ahead and propagate the function name changes into the data client as well. @cprice404 do you have a strong opinion here?

Reply:
I had the same question there.

Reply:
Let's please go ahead and update them.

Reply:
your suggestion makes sense to me

Reply:
> Let's please go ahead and update them.

Roger.